apps/prairielearn/python/prairielearn/sympy_utils.py

```diff
@@ -1,3 +1,5 @@
+"""This file contains utility functions for parsing and evaluating sympy expressions."""
```

Comment:
When not used as a package name, SymPy has different capitalization.

```suggestion
"""Utility functions for parsing and evaluating SymPy expressions."""
```